src/main/flight/imu.c

```diff
@@ -251,27 +248,27 @@ STATIC_UNIT_TESTED void imuMahonyAHRSupdate(float dt, float gx, float gy, float
 
 #ifdef USE_MAG
     // Use measured magnetic field vector
-    fpVector3_t mag_bf = {{mag.magADC[X], mag.magADC[Y], mag.magADC[Z]}};
-    float recipMagNorm = vectorNormSquared(&mag_bf);
+    vector3_t mag_bf = {{mag.magADC[X], mag.magADC[Y], mag.magADC[Z]}};
```

Comment:
(magADC should be refactored to vectrr3 eventually)

Reply:
Done